tools/fetch_elfutils.sh

```diff
@@ -8,13 +8,13 @@ IFS=$'\n\t'
 
 usage() {
     echo "Usage :"
-    echo "$0 <version> <md5> <path> <c-compiler>"
+    echo "$0 <version> <md5> <path> <c-compiler> <extra-c-flags>"
     echo ""
     echo "Example"
-    echo "  $0 0.183 6f58aa1b9af1a5681b1cbf63e0da2d67 ./vendor gcc"
+    echo "  $0 0.183 6f58aa1b9af1a5681b1cbf63e0da2d67 ./vendor gcc -O0"
```

Comment:
If we're taking the `usage()` function seriously as documentation, should probably do one of two things
1. Specify in the online help that the fifth argument can't be split by saying something like `\"-O0 -g\"` with the double-quotes
2. Have the argument be _all except the first four parameters_ (which is a little tricky, since each token will need to be expanded by spaces in case the user does indeed pass `"-O0 -g"` as a single argument)
3.  Detect when there is a 6th parameter and throw an error (forcing better use of the script)

This is somewhat of a nit for now, until we come back to use this later and the globbing behavior makes the debug flags secretly not propagate for one reason or another.

Reply:
true, thanks